Translations/devilutionx.pot

```diff
@@ -3652,7 +3652,6 @@ msgid "knocks target back"
 msgstr ""
 
 #: Source/items.cpp:3619
-#, c-format
```

Comment:
Was this autogenerated or did you manually remove it? If it was done manually then this will just return the next time we update the translation :(

Reply:
Ok done in items.cpp that will propagate it into devilution.pot

Reply:
perfect